pom.xml

```diff
@@ -547,6 +563,56 @@ limitations under the License.
           </plugins>
         </build>
       </profile>
+      <!-- run with the following command -->
```

Comment:
since our pom is already enormous, we'd like to keep it free of run configurations, so please remove

Reply:
I don't know how to run TPT builder from the command line.  I tried figuring out if there is a way to do this by running "mvn exec:java" and specifying the repositories where the lexicons exist but I couldn't figure it out.  I think it is going to be extremely frustrating to build the TPTs without a profile unless there's a way to document how to run a maven command without one.  That's beyond my maven skills.

Reply:
Ok this makes sense to me -- I expect we'll break this out into the separate project anyway, where the pom should be simpler